src/core/constants.ts

```diff
@@ -1,12 +1,21 @@
+export interface Nis1NetworkType {
```

Comment:
why put interface / type in `constants`? Also not sure we need it ??

Reply:
I just want to use `type` on the `BasicNetworkTester` params.
Maybe we need `interface.ts` for all the interface?